### PR TITLE
171 week advance reminder email

### PR DESCRIPTION
**Pull request by NiquiCara**
Ref #171 

Creates mailer for when carrier is due in a week. Tested with 
```
2.6.3 :004 > ReminderMailer.week_advance_notice_email(User.first, "Bloop", "Farts", "Banananananas")
  User Load (0.4ms)  SELECT  "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT $1  [["LIMIT", 1]]
  Rendering reminder_mailer/week_advance_notice_email.html.erb within layouts/mailer
  Rendered reminder_mailer/week_advance_notice_email.html.erb within layouts/mailer (0.5ms)
ReminderMailer#week_advance_notice_email: processed outbound mail in 15.5ms
 => #<Mail::Message:47130221780160, Multipart: false, Headers: <From: from@example.com>, <To: email@example.com>, <Subject: Baby Carrier Due For Return>, <Mime-Version: 1.0>, <Content-Type: text/html>>
```

**merge after #189**

### CODE REVIEW

File: app/mailers/reminder_mailer.rb [hunk 1]
```diff
@@ -7,6 +7,15 @@ def overdue_email(user, carrier_name, current_location, due_date)
     @due_date = due_date
     mail(to: @user.email, subject: 'Baby Carrier Overdue For Return')
   end
+
+  def week_advance_notice_email(user, carrier_name, current_location, due_date)
+    @user = user
+    @first_name = user.full_name
```
Comment (dixpac) [inline app/mailers/reminder_mailer.rb:21]: We don't need this instance variable, we can use `@user` variable to access `full_name` directly in the mailer template

**Review by dixpac — APPROVED** [review]

**Review by dixpac — CHANGES_REQUESTED** [review]
Thanks! Looks good just one tinny comment

**Comment by pollygee** [comment]
@NiquiCara one last time we have that merge conflict, can you please resolve so we can merge this last one?  Thanks!

**Comment by dixpac** [comment]
@NiquiCara  @pollygee  and I resolved conflicts. Thanks so much for your help, keep it coming :)

**Pull request merged**